src/main/webapp/script.js

```diff
@@ -136,6 +136,9 @@ function fetchAggregatedJobsBy(option) {
 }
 
 function setGraphOnLoad(data, title, containerName, graphType) {
+  if (data.length == 1) {
+    return;
+  }
   if (graphType == 'pie') {
```

Comment:
You should use === instead of ==. See discussion [here](https://github.com/googleinterns/step229-2020/pull/55#discussion_r485055016)

Reply:
Ah yes - good point! Solved in 953712c.